### PR TITLE
Homepage, Countries,Browse, Dynamic profile and Country page

### PR DESCRIPTION
**Pull request by jaeyholic**
## Description
Worked on the homepage
Created People's card and country card
Added browse page
Added countries page
Added Profile page

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update

## Checklist:

- [ ] I have updated/added documentation affected by my changes.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: client/pages/index.js [hunk 1]
```diff
@@ -1,78 +1,128 @@
...
-          <Spacing />
-          <HeadingTwo>Know your Statesman</HeadingTwo>
-          <Paragraph><bold>400</bold> Publicly available Manifestos of your elected politicians and statesmen</Paragraph>
-          <InputWrapper width="60%" placeholder="search" />
-          <select placeholder="Country"><option>Ghana</option></select>
-          <Spacing />
-        </section>
-        <Line />
+  React.useEffect(() => {
+    const fetchData = async () => {
```
Comment (koolamusic) [inline client/pages/index.js:21]: Can we have like a `resource.js` or something similar for API so we don't directly call the api like in this place we'd rather do 

```jsx
import {COUNTRY_RESOURCE} from 'path'

Axios.get(COUNTRY_RESOURCE)
```

Feel free to use a naming convention you deem fit

**Review by koolamusic — CHANGES_REQUESTED** [review]
Everything looks good, I left a comment on one of the files, please check and revert

File: client/pages/index.js (hunk 1 above)
Comment (jaeyholic) [reply client/pages/index.js:21]: Okay sure, I will work on that and create a PR for it tomorrow

Comment (koolamusic) [reply client/pages/index.js:21]: Meanwhile we need to work on your PR Titles :laughing:

Comment (jaeyholic) [reply client/pages/index.js:21]: Haha, that's something, I will work on before my next PR :)

**Pull request merged**